FE/src/components/landingPage/WelcomeBlurb.tsx

```diff
@@ -0,0 +1,33 @@
+import Typography from '@foundation/Typography/Typography';
+import { css, keyframes } from '@emotion/react';
+
+const WelcomeBlurb: React.FC = () => {
+  return (
+    <Typography
+      paragraph
+      variant={'title3'}
+      css={css`
+        font-size: 3rem;
+        line-height: 4rem;
+        animation: ${fadeInUp} 1.5s ease forwards;
+      `}
+    >
+      {`로그인 없는 
+간편한 면접 서비스
```

Comment:
요거 정렬이 살짝 이상한데... 왜그럴까요😂
또 eslint가 안먹나...

Reply:
이건 백틱 안에 있는 문자라서 그렇습니다!
정렬을 맞춰서 출력하게되면 글자가 밀리더라구요ㅠ

Reply:
```suggestion
    <Typography
      paragraph
      variant={'title3'}
      css={css`
        font-size: 3rem;
        line-height: 4rem;
        animation: ${fadeInUp} 1.5s ease forwards;
      `}
    >
      로그인 없는
      <br />
      간편한 면접 서비스
      <br />
      곰터뷰
    </Typography>
```

@adultlee @milk717 요거는 어떤가요?